3/README.md

```diff
@@ -1,4 +1,11 @@
-# Dynamically add/remove validators at runtime (Insecure version)
+```
+shortname: 3/UPSERT-VALIDATORS
+name: Dynamically add/remove validators at runtime (Insecure version)
+type: Feature
```

Comment:
I think the type is `Standard track`

Reply:
@ttmc the same was [fixed](https://github.com/bigchaindb/BEPs/blob/master/3/README.md) in seperate PR and merged.

Reply:
Yes I noticed after making that comment. Sorry!